app/workers/follows/update_points_worker.rb

```diff
@@ -9,7 +9,8 @@ def perform(article_id, user_id)
       return unless article && user
 
       adjust_other_tag_follows_of_user(user.id)
-      followed_tag_names = user.cached_followed_tag_names
+      followed_tag_names =
+        user.cached_followed_tag_names + user.cached_antifollowed_tag_names
```

Comment:
I needed both for the spec to become green. @benhalpern Do you maybe have a bit more context on this? Would it e.g. be ok to remove the antifollowed tags and update the expected values in the spec? I was a bit unsure about all of this.